### PR TITLE
Add a simple json parser, to make it work in the backend c++ unittest.

### PR DESCRIPTION
**Pull request by Liang1XZhao**
ehsm_napi.cpp add parseStringData function .

Signed-off-by: Liang1XZhao <liang1x.zhao@intel.com>

### CODE REVIEW

File: core/App/ehsm_napi.cpp [hunk 1]
```diff
@@ -80,6 +80,22 @@ typedef struct {
         retString += "}";
 		return StringToChar(retString);
 	};
+
+    char* parseStringData(char* jsonChar, std::string key) const{
```
Comment (syan10) [inline core/App/ehsm_napi.cpp:84]: the function name not very fitable, for the implementation , it seems just for using to retrieve the "key" value, right?


and who will call it?

Comment (Liang1XZhao) [reply core/App/ehsm_napi.cpp:84]: yes. It simply parses string data, which will be used in unit tests.
You can see it in ehsm_core_test.cpp of the next PR.

Comment (syan10) [reply core/App/ehsm_napi.cpp:84]: just see that in the json struct, there's already has a function named as "void addData(string key, string data)", so why not add another function as "void readData(string key, string data)"

Also, you need to re-submit the PR together with the changes in the unittest code.

**Comment by Liang1XZhao** [comment]
resubmit after repair

**Pull request closed without merging**